README.md

```diff
@@ -3,17 +3,25 @@
 Reserve your meals on MealPal automatically, as soon as the kitchen opens.
 Never miss your favourite MealPal meals again!
 
+## Installation
+
+Install virtualenv with all required dependencies:
+
+```bash
+make venv
+```
+
 ## Help
 
 ```bash
-python mealpy.py --help
+venv/bin/python mealpy/mealpy.py --help
```

Comment:
Not sure if this needs to be stated. But I guess some users may not be python users and not necessarily aware of need to activate virtualenv.

Reply:
Maybe just add the activation step `source venv/bin/activate` above? So that we can simply use `python` in all the command examples.

Reply:
I updated instructions.